feature-utils/poly-look/src/react-components/banners/notificationBanner.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+import "./notificationBanner.css";
+
+import standard from "../../static/images/icons/xmark-standard.svg";
+import success from "../../static/images/icons/xmark-success.svg";
+import error from "../../static/images/icons/xmark-error.svg";
+import warning from "../../static/images/icons/xmark-warning.svg";
+
+export const notificationTypes = {
+  standard: { class: "standard", icon: standard },
```

Comment:
If the `notificationTypes` is to prevent typos causing bugs, I would split it. Meaning it would look something like:
`export const notificationsTypes={standard:"standard"}` and something like `const icons = {standard}`. 
From the perspective of using this component you don't care about `class` or `icon` it's more like an internal configuration.

Reply:
If the icon gets the same name as the notification, the string can be mapped to the file name easily too.

Reply:
@g3o314 Please review the new changes and let me know if it looks better now or if more changes are needed 😄 Thanks!

Reply:
@mteresa-jimenez it does :D Thank you!